site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -61,23 +55,15 @@ manifest:
 }
 ```
 
-An instance of the `devtools_page` specified in your extension's manifest is created for every
-DevTools window opened. The page may add other extension pages as panels and sidebars to the
-DevTools window using the [`devtools.panels`][api-panels] API.
-
-{% Aside %}
-
-The `devtools_page` field must point to an HTML page. The DevTools
-page must be local to your extension, so it is best to specify it using a relative URL.
-
-{% endAside %}
+The `devtools_page` field must point to an HTML page. Because the DevTools
+page must be local to your extension, we recommend specifying it it using a relative URL.
 
 The `chrome.devtools.*` API modules are available only to the pages loaded within the DevTools
```

Comment:
```suggestion
The members of the `chrome.devtools` API are available only to the pages loaded within the DevTools
```

Reply:
If you run into any more API names done with wild cards at the end, please remove them.